src/Command/Backup/BackupListCommand.php

```diff
@@ -54,17 +66,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $table->replaceDeprecatedColumns(['created' => 'created_at', 'name' => 'id'], $input, $output);
         $table->removeDeprecatedColumns(['progress', 'state', 'result'], '[deprecated]', $input, $output);
 
-        $headers = [
-            'created_at' => 'Created',
-            'expires_at' => 'Expires',
-            'id' => 'Backup ID',
-            'status' => 'Status',
-            'commit_id' => 'Commit ID',
-            'safe' => 'Safe',
-            'restorable' => 'Restorable',
-            'index' => 'Index',
-            '[deprecated]' => '[Deprecated]',
-        ];
+        $header = $this->tableHeader;
+        $header['safe'] = 'Safe';
+        $header['[deprecated]'] = '[Deprecated]';
         $defaultColumns = ['created_at', 'id', 'restorable'];
```

Comment:
Should the default columns be documented too? Either some shorthand like:

```
Available columns: "created_at"*, "expires_at", "id"*, "status", "commit_id", "live",
"restorable"*, "index", "automated" (* = default columns).
```

or longhand:

```
Available columns: "created_at", "expires_at", "id", "status", "commit_id", "live",
"restorable", "index", "automated".
Default columns: "created_at", "id", "restorable".
```